src/app/app.component.spec.ts

```
@@ -1,28 +1,122 @@
+  let spenderOnboardingService: jasmine.SpyObj<SpenderOnboardingService>;
+  let footerService: jasmine.SpyObj<FooterService>;
+  let tasksService: jasmine.SpyObj<TasksService>;
+  let router: jasmine.SpyObj<Router>;
+  let trackingService: jasmine.SpyObj<TrackingService>;
+  let navController: jasmine.SpyObj<NavController>;
+
+  beforeEach(waitForAsync(() => {
+    platformReadySpy = Promise.resolve();
+    platformSpy = jasmine.createSpyObj('Platform', { ready: platformReadySpy });
+    platformSpy.backButton = {
+      subscribeWithPriority: (priority: number, callback: (processNextHandler: () => void) => Promise<any> | void) => {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)